src/convertToObject.js

```diff
@@ -6,7 +7,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .reduce((prev, el) => {
+      const [property, value] = el
+        .split(':')
+        .map((elem) => elem.trim());
+
+      prev[property] = value;
```

Comment:
The variable naming `prev` and `el` could be improved for better readability. Consider naming `prev` as `accumulator` or `stylesObject` and `el` as `declaration` to more accurately describe their roles in the `reduce` function.